incuna_test_utils/factories/images.py

```diff
@@ -0,0 +1,83 @@
+from io import BytesIO
+
+import factory
+from PIL import Image
+
+
+TEST_SERVER = 'http://testserver'
```

Comment:
This is quite handy in itself. We often use this in projects. Perhaps it should be defined somewhere more accessible.